src/grid/ctrl.coffee

```diff
@@ -53,37 +61,46 @@ class GridCtrl
   ###
   init: ->
     @_headView
-      .setTags( null, 'TH' )
       .setTarget( @element.find( @HEADER_ELEMENT ) )
+      .setTags( null, 'TH' )
     @_rowsView
       .setTarget( @element.find( @BODY_ELEMENT ) )
+    @_pagination
+      .on( @_pagination.CHANGE, @updateRows )
+      .init()
+    @_datasource = new DatasourceArray()
+      .data( @_config.data )
 
-    datasource = new GridDatasourceArray()
-    datasource.data( @_config.data )
-    @setDatasource( datasource )
-
-    return @update()
+    # Count is obtain to config pagination
+    @_datasource.count()
+      .then ( count ) =>
+        @_pagination.setCount( count )
 
   ###
-   * Updates element with data rows
+   * Updates grid element with data rows
    * @return {Promise}
   ###
-  update: ->
+  updateRows: =>
+    if @_config.pagination isnt false
```

Comment:
This will be translated to ```this._config.pagination !== false```. Wouldn't it be enough with ```!this._config.pagination```? Or you want the condition to be strict?